tests/testthat/test-parallel.R

```diff
@@ -1,5 +0,0 @@
-test_that("free_cores works as expected", {
-  skip_on_cran()
-
-  expect_silent(free_cores())
```

Comment:
Please don't delete already, instead please check that the deprecation warning works.
(because the function is still in the package we still need to test it)

Reply:
replaced with deprecation test